frontend/pages/game_page.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tetris</title>
+    <link rel="stylesheet" type="text/css" href="game_page.css">
+
+</head>
+<body>
+    <h1>
+        <span class="red">T</span>
+        <span class="green">E</span>
+        <span class="blue">T</span>
+        <span class="magenta">R</span>
+        <span class="purple">I</span>
+        <span class="orange">S</span>
+    </h1>
+
+    <h2>
+        Your Score is: 
+    </h2>
+    <div class="canvas">
+        <canvas id="myCanvas" width="500" height="700" style="border:1px solid #000000;"></canvas>
```

Comment:
Just thinking if we want the grid to be 10x20 of squares then the ratio of width to height should also be 10:20. e.g right now it's 5:7 so we should find a width/height we like and then match the height/width so it fits the ratio